scripts/tomcat/tomcat8/ubuntu/tomcat8_install.sh

```diff
@@ -122,12 +122,12 @@ cp ${SCRIPT_HOME}/etc/default/${TOMCAT_NAME} /etc/default
 
 if ! grep -i --quiet 'ingest/processed' ${TOMCAT_CONFIG_HOME}/server.xml; then
     echo "Adding Tomcat context path for tile images..."
-    sudo sed -i.bak 's@<\/Host>@  <Context docBase=\"'"$HOOT_HOME"'\/ingest\/processed\" path=\"\/static\" \/>\n      &@' ${TOMCAT_CONFIG_HOME}/server.xml
+    sudo sed -i.bak 's@<\/Host>@  <Context docBase=\"'"$HOOT_HOME"'\/userfiles\/ingest\/processed\" path=\"\/static\" \/>\n      &@' ${TOMCAT_CONFIG_HOME}/server.xml
```

Comment:
VagrantProvision.sh always runs tomcat8_install.sh, which will overwrite any previous version of context.xml and server.xml files under Tomcat's conf folder.